src/libs/ReportUtils.ts

```diff
@@ -4373,7 +4373,7 @@ function buildOptimisticGroupChatReport(participantAccountIDs: number[], reportN
         undefined,
         undefined,
         undefined,
-        CONST.REPORT.NOTIFICATION_PREFERENCE.HIDDEN,
+        CONST.REPORT.NOTIFICATION_PREFERENCE.ALWAYS,
```

Comment:
default value is ALWAYS
```suggestion
        undefined,
```

This may also require backend update.